### ISSUE TITLE
[Feature Request] Add the posibility to turn off price refreshing.

### ISSUE THREAD

**Issue by KristofMikle** [viktorgullmark/exilence-next#708]
Hi!

Currently when I press the Fetch Snapshot Button it fetches what I have in my stashes, and also updates the price of items from poe.ninja. I would like to have the option to just fetch the new content of my stash tabs without fetching the new prices from poe.ninja. 

I think this could be done with either a new button, or maybe a new checkbox in options/settings.

My problem is that while I am mapping, I like to fetch after every map to see how much money I made in the map. But often this is not accurate because of the changing price of items previously in my stash.

**Comment by Tomagathericon** [comment]
Just writing to say I would also like to see this feature. Being able to turn off automatic price updates would really help gauge net gains / losses more effectively without having to worry about fluctuation ruining your numbers.

**Comment by arnoudvanderleer** [comment]
Yeah! Just enable switching between auto-fetching prices and fetching prices with a button, just like with snapshots.

Also, currently, when new prices have been fetched, the change in net worth is included in the next stash snapshot. I am not sure about this, but maybe it would be nice if you create separate snapshots for price fetches and for stash tab fetches?
